content/build/building/variables.md

```diff
@@ -0,0 +1,519 @@
+| [BUILDX_CONFIG](#buildx_config)                                             | String            | Specify location for configuration, state, and logs. |
+| [BUILDX_EXPERIMENTAL](#buildx_experimental)                                 | Boolean           | Turn on experimental features.                       |
+| [BUILDX_GIT_CHECK_DIRTY](#buildx_git_check_dirty)                           | Boolean           | Enable dirty Git checkout detection.                 |
+| [BUILDX_GIT_INFO](#buildx_git_info)                                         | Boolean           | Remove Git information in provenance attestations.   |
+| [BUILDX_GIT_LABELS](#buildx_git_labels)                                     | String \| Boolean | Add Git provenance labels to images.                 |
+| [BUILDX_NO_DEFAULT_ATTESTATIONS](#buildx_no_default_attestations)           | Boolean           | Turn off default provenance attestations.            |
+| [BUILDX_NO_DEFAULT_LOAD](#buildx_no_default_load)                           | Boolean           | Turn off loading images to image store by default.   |
+| [EXPERIMENTAL_BUILDKIT_SOURCE_POLICY](#experimental_buildkit_source_policy) | String            | Specify a BuildKit source policy file.               |
+
+BuildKit also supports a few additional configuration parameters. Refer to
+[BuildKit built-in build args](../../engine/reference/builder.md#buildkit-built-in-build-args).
+
+You can express Boolean values for environment variables in different ways.
+For example, `true`, `1`, and `T` all evaluate to true.
+Evaluation is done using the `strconv.ParseBool` function in the Go standard library.
+See the [reference documentation](https://pkg.go.dev/strconv#ParseBool) for details.
+
+### BUILDKIT_COLORS
+
+Changes the colors of the terminal output. Set `BUILDKIT_COLORS` to a CSV string
+in the following format:
+
+```console
+$ export BUILDKIT_COLORS="run=123,20,245:error=yellow:cancel=blue:warning=white"
+```
+
+Color values can be any valid RGB hex code, or one of the
+[BuildKit predefined colors](https://github.com/moby/buildkit/blob/master/util/progress/progressui/colors.go).
+
+Setting `NO_COLOR` to anything turns off colorized output, as recommended by
+[no-color.org](https://no-color.org/).
+
+### BUILDKIT_HOST
+
+_Introduced in [Buildx v0.9.0](../release-notes.md#090)_
+
+You use the `BUILDKIT_HOST` to specify the address of a BuildKit daemon to use
+as a remote builder. This is the same as specifying the address as a positional
+argument to `docker buildx create`.
+
+Usage:
+
+```console
+$ export BUILDKIT_HOST=tcp://localhost:1234
+$ docker buildx create --name=remote --driver=remote
+```
+
+If you specify both the `BUILDKIT_HOST` environment variable and a positional
+argument, the argument takes priority.
+
+### BUILDKIT_PROGRESS
+
+Sets the type of the BuildKit progress output. Valid values are:
+
+- `auto` (default)
+- `plain`
+- `tty`
```

Comment:
Missing `rawjson` related to https://github.com/docker/buildx/pull/2035#issuecomment-1935871933